sn_protocol/src/messages/register.rs

```diff
@@ -16,47 +16,6 @@ use sn_registers::Register;
 use serde::{Deserialize, Serialize};
 use xor_name::XorName;
 
-/// [`Register`] read operations.
-#[derive(Hash, Eq, PartialEq, PartialOrd, Clone, Serialize, Deserialize, Debug)]
-pub enum RegisterQuery {
-    /// Retrieve the [`Register`] at the given address.
-    ///
-    /// This should eventually lead to a [`GetRegister`] response.
-    ///
-    /// [`GetRegister`]: QueryResponse::GetRegister
-    Get(RegisterAddress),
-    /// Retrieve the current entries from the [`Register`] at the given address.
-    ///
-    /// Multiple entries occur on concurrent writes. This should eventually lead to a
-    /// [`ReadRegister`] response.
-    ///
-    /// [`ReadRegister`]: QueryResponse::ReadRegister
-    Read(RegisterAddress),
```

Comment:
Are we not here removing a lot of our register API surface? 

Might we want to replicate these at the client if they're being removed at the protocol level?

Reply:
not sure, but so far they are only used when transmission messages,
and as clippy complains shows, none of them is required any more after switched to `kad::record` approach.